lib/profile/common/input_field.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class InputField extends StatelessWidget {
+  final bool enabled;
+  final String labelText;
+  final Widget formField;
+
+  InputField({
+    Key key,
+    this.enabled,
+    @required this.labelText,
+    @required this.formField
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.only(bottom: 10.0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          buildFieldLabel(),
+          formField
+        ]
+      )
+    );
+  }
+
+  Widget buildFieldLabel() {
+    return Container(
+      padding: EdgeInsets.all(0.0),
+      child: Text(
+        labelText,
+        style: GoogleFonts.muli(
+          textStyle: TextStyle(
+            color: (enabled == null || enabled) ? Colors.black : Colors.black38, 
```

Comment:
So in this widget, `enabled` is an optional boolean prop/parameter.
`enabled == null` is when the prop is undefined, while `enabled` is when its value is set to `true`.
 I only define this prop for the `End year`/`End date` fields since they need to be disabled in some cases. One example is that `End year`/`End date` fields are initially disabled until the user chooses a `Start year`/`Start date`.